src/resources/InsightPanelInterfaces/InsightPanelInterface.ts

```diff
@@ -1,12 +1,13 @@
 import API from '../../APICore';
 import {New, PageModel} from '../BaseInterfaces';
+import {IListHostedInterfacesParameters} from '../HostedInterfacesCore';
 import Resource from '../Resource';
-import {IListInsightPanelInterfacesParameters, InsightPanelInterfaceConfiguration} from './InsightPanelInterface.model';
+import {InsightPanelInterfaceConfiguration} from './InsightPanelInterface.model';
 
 export default class InsightPanelInterface extends Resource {
     static baseUrl = `/rest/organizations/${API.orgPlaceholder}/insightinterface/v1/interfaces`;
 
-    list(options?: IListInsightPanelInterfacesParameters) {
+    list(options?: IListHostedInterfacesParameters) {
```

Comment:
nitpik: naming-wise we don't really use `IPotatoTomato`.

If I check other resources we tend to call List Options as such `ListNameOfMyResourcesParams`.

So here it'll be `ListHostedInterfacesParams` (so yeah just drop the I and shorten a bit and call it a day)